cg/utils/calculations.py

```diff
@@ -1,6 +1,11 @@
 """Module to hold functions for calculations."""
 
 
+def divide_by_thousand_with_one_decimal(number: float | int) -> float:
+    """Divide a given number by a thousand."""
```

Comment:
Prime example of an unnecessary doc string according to me 😉

Reply:
true

Reply:
fixed